### ISSUE TITLE
[Outreachy Task Submission] Accessibility Issue with the 'show password' button value text in login/signup screens

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4757]
# Intro
This is one of two accessibility issues I found with the 'Forgotten Password?' button in the login screen. Close to, but still very different from #4766. 

## Issue
The button for making passwords visible has no value text. ***This is a violation of WCAG Success Criterion 1.1.1 (Level A) - Non-text content***. [See](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

## Description.
***Success Criterion 1.1.1 (Level A) - Non-text content*** requires that all non-text content presented to users has a text alternative that serves the equivalent purpose, except for situations where the content is intended to be omitted by the author or is decorative in nature.

## Impact
Accessibility software that can either convert text to visuals (for deaf people), and braille (for blind people) will not work. People who cannot understand the meaning of images/icons would struggle with this. Additionally, there would be no way to search for visual content in a text-based manner. 

## Fix
Add a value text to the button. I will raise a PR for this, shortly.

**Comment by Chiemezuo** [comment]
@Angamanga Could you help me with the "**Outreachy Task Submission**" label?
